newm-shared/src/main/kotlin/io/newm/shared/ktx/BigDecimalExt.kt

```diff
@@ -1,5 +1,15 @@
 package io.newm.shared.ktx
 
 import java.math.BigDecimal
+import java.text.NumberFormat
+import java.util.Locale
+
+fun BigDecimal.toGroupingString(): String {
+    val formatter = NumberFormat.getInstance(Locale.US).apply {
```

Comment:
`NumberFormat` is not thread-safe. I would use a `ThreadLocal` similary to how it works in `DMYLocalDateSerializer` to ensure this doesn't cause problems for us.